BetSystem/Contract/BetOnEventDto.cs

```diff
@@ -9,5 +10,13 @@ public record BetOnEventDto
         public int Amount { get; set; }
         public int Interest { get; set; }
         public int AmountToPay { get; set; }
+        public int TeamId { get; set; }
+        public int EventId { get; set; }
+
+        public void UpdateAmountToPay()
```

Comment:
I think that its must be in separate class/service. We define DTO - Data Transfer Object and it's purpose to transfer data between processes/logic and I think it must not include logic. Also it can be tested.

Reply:
Deleted because it was usless